### DISCUSSION TITLE
pass a connection settings to postgresql in Prisma

### DISCUSSION THREAD

**Discussion by mdespriee** [prisma/prisma#7378] [Q&A] [answered]
I need to set a settings in a postgresql session (keepalives_idle). It needs to be set only once, at connection.

It does not seem to be passed and is not documented in the list in Prisma docs.

What's the proper way to do it in Prisma ?

**Comment by janpio** [comment, marked as answer]
We do not have any mechanism to execute a query per connection yet. Best open a feature request issue and describe your use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
